redbot/core/events.py

```diff
@@ -266,16 +266,17 @@ async def on_command_error(ctx, error, unhandled_by_cog=False):
                 "Exception in command '{}'".format(ctx.command.qualified_name),
                 exc_info=error.original,
             )
-
-            message = _(
-                "Error in command '{command}'. Check your console or logs for details."
-            ).format(command=ctx.command.qualified_name)
             exception_log = "Exception in command '{}'\n" "".format(ctx.command.qualified_name)
             exception_log += "".join(
                 traceback.format_exception(type(error), error, error.__traceback__)
             )
             bot._last_exception = exception_log
-            await ctx.send(inline(message))
+
+            message = (
+                await bot._config.invoke_error_msg()
+                or "`Error in command {command}. Check your console or logs for details.`"
+            )
```

Comment:
You didn't actually mark the string as translatable. I would also suggest to make the default message consistent with the one in Trivia cog:
```suggestion
            message = await bot._config.invoke_error_msg()
            if not message:
                if ctx.author.id in bot.owner_ids:
                    message = inline(_("Error in command {command}. Check your console or logs for details."))
                else:
                    message = inline(_("Error in command {command}."))
```